ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -204,6 +247,56 @@ private void terminateCluster(final Command command) {
     log.info("The KSQL server was terminated.");
   }
 
+  public double checkCommandRunnerStatus() {
+    if (currentCommand == null) {
+      return 1;
+    }
+
+    return Duration.between(currentCommand.right, Instant.now()).toMillis()
+        < COMMAND_RUNNER_HEALTH_TIMEOUT.toMillis() ? 1 : 0;
+  }
+
+  private void configureCommandRunnerStatusMetric() {
+    final String metricName = "liveness-indicator";
+    final String description =
+        "A metric indicating the status of the commandRunner. "
+            + "If value 1, the commandRunner is processing commands normally."
```

Comment:
If we define the enum you suggested, what's the advantage of emitting the metric as a string rather than an integer? I'm not sure how well datadog (and other similar tools) plays with string-valued metrics.

Reply:
It should be ok I think. Wouldn't using strings be similar to how a query's status is tracked by string values?

Reply:
The advantage is that it makes more sense if you're just dumping the JMX. The datadog agent lets you configure a mapping from a string to a numerical metric. The disadvantage there is that you have to keep the mapping in the dd config in sync with your code (so if you add a new status you have to update your config). I'm leaning toward emitting a numerical metric for that reason. I think we should still implement it as an enum though in our code.